web/yo/app/scripts/controllers/gene.js

```diff
@@ -2371,7 +2371,12 @@ angular.module('oncokbApp')
                 if ($scope.reviewMode) {
                     return false;
                 }
-                if (type === 'mutation_effect') {
+                if (type === 'mutation') {
+                    if (obj.mutation_effect.oncogenic || obj.mutation_effect.effect || obj.mutation_effect.description || obj.mutation_effect.short || !_.isUndefined(obj.tumors)) {
```

Comment:
`obj.tumors` could be defined but an empty array?

Reply:
No. Firebase will auto delete empty object and array.